raiden/api/rest.py

```diff
@@ -80,9 +80,18 @@ def normalize_events_list(old_list):
     return new_list
 
 
-def jsonify_with_response(data, status_code):
+def api_response(result, status_code=httplib.OK):
     response = make_response((
-        json.dumps(data),
+        json.dumps(result),
+        status_code,
+        {'mimetype': 'application/json', 'Content-Type': 'application/json'}
+    ))
+    return response
+
+
+def api_error(errors, status_code=httplib.CONFLICT):
```

Comment:
I suggest to make `status_code` mandatory and also check for the given `status_code` to be in our (known) set of error codes:
```
ERRORS = [httplib.CONFLICT, httplib.REQUEST_TIMEOUT, ...]
def api_error(errors, status_code):
  assert status_code in ERRORS, "programming error"
  ...
```

Reply:
okay willl do